lib/reverse_sentence.rb

```diff
@@ -1,6 +1,38 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^3), cubic, because there are three while loops within reverse_sentence() and reverse() that are nested. The amount of iterations may increase proportionately to the data being passed through.
```

Comment:
Since the 1st two are not nested this is actually O(n + nm), but because each word is pretty small (that's an assumption) you could reduce this to O(n).